pkg/devspace/configure/image.go

```diff
@@ -136,65 +137,73 @@ func (m *manager) AddImage(imageName, image, dockerfile, contextPath string, doc
 		m.log.WriteString("\n")
 		m.log.Info("DevSpace does *not* require pushing your images to a registry but let's assume you wanted to do that (optional)")
 
+		registryOptions := []string{skipRegistry, useDockerHub, useGithubRegistry, useOtherRegistry}
+		isLoggedIntoRegistry := false
 		authConfig, err := dockerClient.GetAuthConfig(dockerHubHostname, true)
 		if err == nil && authConfig.Username != "" {
 			useDockerHub = useDockerHub + fmt.Sprintf(registryUsernameHint, authConfig.Username)
 			registryDefaultOption = useDockerHub
+			registryOptions = []string{useDockerHub, useGithubRegistry, useOtherRegistry, skipRegistry}
+			isLoggedIntoRegistry = true
 		}
 
-		authConfig, err = dockerClient.GetAuthConfig(githubContainerRegistry, true)
-		if err == nil && authConfig.Username != "" {
-			useGithubRegistry = useGithubRegistry + fmt.Sprintf(registryUsernameHint, authConfig.Username)
+		if !isLoggedIntoRegistry {
```

Comment:
While this works, it would not show the github user if you are logged in dockerhub and github at the same time

Reply:
So here we want to see the logged-in username and also if a user is logged into docker and Github at the same time we want to make docker as a default value, correct?

Reply:
@pratikjagrut yes exactly, there are essentially 4 cases here:

1. Case: User is logged into Dockerhub & Github: Dockerhub Option default and show usernames of Dockerhub and github
2. Case: User is logged into Dockerhub: Dockerhub option default and show username of Dockerhub
3. Case: User is logged into Github: Github option default and show username of Github
4. Case: User is not logged into Dockerhub or Github: Skip registry selected